pkg/releaselock/releaselock.go

```diff
@@ -0,0 +1,89 @@
+package releaselock
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"net"
+	"net/rpc"
+	"strconv"
+
+	"github.com/prashanthpai/sunrpc"
+	"k8s.io/klog/v2"
+)
+
+const (
+	pmapProgramNumber  = uint32(100021)
+	pmapProgramVersion = uint32(4)
+	programNumber      = uint32(200002)
+	programVersion     = uint32(1)
+	procedureNumber    = uint32(1)
+	procedureName      = "IN_BAND_PROPRIETARY_LOCK_OPS_PROG.IN_BAND_PROPRIETARY_LOCK_OPS_V1"
+	rpcPort            = "111"
+	protocol           = "tcp"
+)
+
+type releaseLockResponse struct {
+	status releaseLockStatus
+}
+
+type releaseLockStatus uint
+
+// ReleaseLock calls the NFS server to remove all advisory locks on the given GKE node.
```

Comment:
ReleaseLock calls the Filestore server to remove all advisory locks for a given GKE node IP.